can/interfaces/vector/canlib.py

```diff
@@ -217,19 +242,19 @@ def __init__(
             if fd:
                 self.canFdConf = xlclass.XLcanFdConf()
                 if bitrate:
-                    self.canFdConf.arbitrationBitRate = ctypes.c_uint(bitrate)
+                    self.canFdConf.arbitrationBitRate = arg_to_c_uint(bitrate)
                 else:
-                    self.canFdConf.arbitrationBitRate = ctypes.c_uint(500000)
-                self.canFdConf.sjwAbr = ctypes.c_uint(sjwAbr)
-                self.canFdConf.tseg1Abr = ctypes.c_uint(tseg1Abr)
-                self.canFdConf.tseg2Abr = ctypes.c_uint(tseg2Abr)
+                    self.canFdConf.arbitrationBitRate = arg_to_c_uint(500000)
```

Comment:
I don’t think you need to convert it to a ctypes type explicitly since the type of the attribute has already been specified so ctypes will do it for you. Converting to int would suffice.

Reply:
agree